core/src/blocks/chat.rs

```diff
@@ -313,6 +313,9 @@ impl Block for Chat {
                 if let Some(Value::String(s)) = v.get("openai_organization_id") {
                     extras["openai_organization_id"] = json!(s.clone());
                 }
+                if let Some(Value::String(s)) = v.get("openai_hostname") {
+                    extras["openai_hostname"] = json!(s.clone());
```

Comment:
Let's have a bit more control here if only for security reasons (if someone takes control of that they can reroute our entire traffic).

Let's instead use a binary flag openai_eu_region true/false